src/solver/conv/conv_ocl_dir2D_bwdWrW_2.cpp

```diff
@@ -461,6 +463,18 @@ template <int N_BATCH_LOOPS>
 bool ConvOclBwdWrW2<N_BATCH_LOOPS>::IsApplicableBase(const ExecutionContext& ctx,
                                                      const ProblemDescription& problem) const
 {
+    // Disable this solver due to random GPU memory access faults on gfx11 and gfx12
```

Comment:
It's better to place the description above the corresponding #define WORKAROUND_...

Reply:
I put separate comment above #define macros